api/tests/core/reminders/external/test_reminders_notifications.py

```diff
@@ -0,0 +1,125 @@
+
+        offer_2 = offer_factories.OfferFactory(isActive=False)
+        publication_date_2 = datetime.utcnow() - timedelta(minutes=10)
+        offer_factories.FutureOfferFactory(offer=offer_2, publicationDate=publication_date_2)
+
+        offer_3 = offer_factories.OfferFactory(isActive=False)
+        publication_date_3 = datetime.utcnow() - timedelta(minutes=15)
+        offer_factories.FutureOfferFactory(offer=offer_3, publicationDate=publication_date_3)
+
+        with patch(
+            "pcapi.core.reminders.external.reminders_notifications.notify_users_future_offer_activated"
+        ) as notify_users_future_offer_activated_mock:
+            notify_users_for_future_offers_activations()
+            notify_users_future_offer_activated_mock.assert_has_calls(
+                [call(offer=offer), call(offer=offer_2)], any_order=True
+            )
+
+
+@pytest.mark.usefixtures("db_session")
+class NotifyUsersFutureOfferActivatedTest:
+    def test_notify_users_future_offer_activated(self):
+
+        user_1 = users_factories.UserFactory()
+        user_2 = users_factories.UserFactory()
+        user_3 = users_factories.UserFactory()
+
+        offer = offer_factories.EventOfferFactory(isDuo=True, name="Super Future Offer")
+        future_offer = offer_factories.FutureOfferFactory(offer=offer)
+        offer_2 = offer_factories.OfferFactory()
+        future_offer_2 = offer_factories.FutureOfferFactory(offer=offer_2)
+
+        factories.FutureOfferReminderFactory(futureOffer=future_offer, user=user_1)
+        factories.FutureOfferReminderFactory(futureOffer=future_offer, user=user_2)
+        factories.FutureOfferReminderFactory(futureOffer=future_offer_2, user=user_3)
+
+        notify_users_future_offer_activated(offer)
+
+        expected_push_counts = 1  # for trigger event future offer activated
+        assert len(push_testing.requests) == expected_push_counts
+
+        offer_attributes = {
+            "offer_id": offer.id,
+            "offer_name": "Super Future Offer",
+            "offer_category": "CINEMA",
+            "offer_subcategory": "SEANCE_CINE",
+            "offer_type": "duo",
+        }
+        expected_payload = [
+            {
+                "user_id": user_1.id,
+                "events": [
+                    {
+                        "name": "ue.Future_offer_activated",
+                        "attributes": offer_attributes,
+                    }
+                ],
+            },
+            {
+                "user_id": user_2.id,
+                "events": [
+                    {
+                        "name": "ue.Future_offer_activated",
+                        "attributes": offer_attributes,
+                    }
+                ],
+            },
+        ]
+        future_offer_activated_event = next(event for event in push_testing.requests)
+        event_payload = future_offer_activated_event["payload"]
```

Comment:
```suggestion
        event_payload = push_testing.requests[0]["payload"]
```
(low / subjectif) Je trouve ça plus concis et facile à lire